lib/ghc-mod/ghc-modi-process.coffee

```diff
@@ -340,6 +340,27 @@ class GhcModiProcess
           ]
         replacement: text
 
+  doHoleFill: (buffer, crange) =>
+    return Promise.resolve [] unless buffer.getUri()?
+    crange = Util.tabShiftForRange(buffer, crange)
+    @queueCmd 'typeinfo',
+      interactive: @caps?.interactiveCaseSplit ? false
```

Comment:
I think it's safe to use `interactive: true` here. `interactiveCaseSplit` is a workaround for a ghc-mod bug where `case-split` command crashed interactive mode, but I don't think it's the case with `auto`.

NB: `sig-fill` just crashed in most cases since ghc-mod-5.3, and was finally fixed in 5.6 IIRC, so the same workaround is applied there to avoid crashing interactive mode.